src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "systemtask/SystemTask.h"
+#include "components/motion/MotionController.h"
+#include "components/motor/MotorController.h"
+
+#include <array>
+
+namespace Pinetime::Applications::Screens {
```

Comment:
This shows a warning `nested namespace definition is a C++17 extension`

Reply:
Fixed this in https://github.com/JF002/InfiniTime/pull/565/commits/75c7e280842f08b91be91de121d600ec5f87defd